src/views/AnnualReport.vue

```diff
@@ -405,7 +392,7 @@ export default class AnnualReport extends Vue {
   private agmDate = null
   private agmExtension = null
   private noAgm = null
-  private agmDateValid = false
+  protected agmDateValid = false
```

Comment:
I'm not sure why, but Volar (extension) stopped highlight a ton of warnings when I changed just this one.

It doesn't really matter anyway because the template can see everything in the script: access specifiers are ignored at the moment.